sdv/logging/sdv_logger_config.yml

```diff
@@ -4,24 +4,28 @@ loggers:
   SingleTableSynthesizer:
     level: INFO
     propagate: false
+    format: CSV
```

Comment:
the standard config for logging has the `formatter` key word which can be specified under the handler section. https://docs.python.org/3/library/logging.config.html#dictionary-schema-details